policy/cnspec_policy.proto

```diff
@@ -422,6 +542,23 @@ message ReportCollection {
   map<string, ResolvedPolicy> resolved_policies = 5;
 }
 
+message ComplianceReport {
```

Comment:
should this be named `ComplianceReport` to be consistent?

Reply:
Great idea to naming it `FrameworkReport` i'll update it